providers/pipedrive/write.go

```diff
@@ -16,21 +17,33 @@ type writeResponse struct {
 // Write creates or updates records in a pipedriver account.
 // https://developers.pipedrive.com/docs/api/v1
 func (c *Connector) Write(ctx context.Context, config common.WriteParams) (*common.WriteResult, error) {
+	var write common.WriteMethod
```

Comment:
I wouldn't mix the working of both modules. Can you look at how the google connector is written (or any multi-module provider that @Cobalt0s has written)? That has an approach that we can copy here